Libraries/LibJS/Runtime/PrimitiveString.cpp

```diff
@@ -83,9 +74,7 @@ String PrimitiveString::utf8_string() const
     resolve_rope_if_needed(EncodingPreference::UTF8);
 
     if (!has_utf8_string()) {
-        if (has_byte_string())
-            m_utf8_string = MUST(String::from_byte_string(*m_byte_string));
-        else if (has_utf16_string())
+        if (has_utf16_string())
```

Comment:
nit: Would look nicer IMO for this to match `PrimitiveString::utf16_string` below:
```c++
if (!has_utf8_string()) {
    VERIFY(has_utf16_string());
    m_utf8_string = m_utf16_string->to_utf8();
}
```